src/eos/eosEngine.js

```diff
@@ -420,33 +512,13 @@ export class EosEngine extends CurrencyEngine {
   }
 
   // synchronous
```

Comment:
We should ditch these incorrect synchronous / asynchronous comments, now that the `async` keyword exits to convey the same info in a computer-enforced manner.